src/calculateRentalCost.js

```diff
@@ -1,10 +1,26 @@
+'use strict';
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const SHORT_TERM = 3;
+
+  let price = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    price -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    price -= SHORT_TERM_DISCOUNT;
```

Comment:
Consider using `else if (days >= SHORT_TERM && days < LONG_TERM)` to make the condition more explicit and avoid any potential confusion about overlapping conditions.